docs/setup/vbmh-setup.md

```diff
@@ -85,7 +85,7 @@ EOF
 Add this line to /etc/hosts
 
 ```text
-192.168.124.99 boot.ironic.suse.baremetal api.ironic.suse.baremetal inspector.ironic.suse.baremetal media.suse.baremetal
+192.168.125.100 boot.ironic.suse.baremetal api.ironic.suse.baremetal inspector.ironic.suse.baremetal media.suse.baremetal
```

Comment:
I think we can remove the boot/api/inspector names here, since it's only used to reference the cached image on the network-infra VM?

Reply:
Yes, currently we use it only to resolve the `media` so I will remove it